docs/visualize/lens.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[lens]]
+== Lens
+
+beta[]
+
+*Lens* provides you with a simple way to create basic visualizations.
+
+With Lens, you can:
+
+* Easily make sense of your data with basic visualizations.
+* Use aggregations to plot any data set, no matter how large.
+* Focus the data you want to display with searches.
+* Add your visualizations to Dashboards and Discover.
+
+//TODO For an overview of Lens, watch <<Lens video, Getting started with Lens>>.
+
+[float]
+[[choose-lens-visualization]]
+=== Types of Lens visualizations
+
+Lens comes with these types of visualizations:
+
+Bar:: Display numerical comparisons across multiple categories. In bar charts,
+data is displayed using a number of bars that represent a particular category.
++
+[role="screenshot"]
+image::images/lens_bar_chart.png[Lens bar chart]
+
+Stacked Bar:: Display numerical comparisons across multiple categories, with the
+added ability to compare the different parts of a particular category. In
+stacked bar charts, each bar represents a particular category, and the segments
+of the bar represent the sub categories of that bar.
++
+[role="screenshot"]
+image::images/lens_stacked_bar_chart.png[Lens stacked bar chart]
+
+Line:: Display data trends over time. In line charts, data is displayed as a
+series of data points that are connected by line segments.
++
+[role="screenshot"]
+image::images/lens_line_chart.png[Lens line chart]
+
+Area:: Display one or more quantities over time. In area charts, points are
+plotted and connected by lines to show the value of a quantity at several
+different times.
++
+[role="screenshot"]
+image::images/lens_area.png[Lens area chart]
+
+Stacked Area:: Display the evolution of numeric variables for several
+categories stacked on top of each other.
++
+[role="screenshot"]
+image::images/lens_stacked_area.png[Lens stacked area chart]
+
+Data Table:: Display your raw data or aggregation results in a tabular format.
++
+[role="screenshot"]
+image::images/lens_data_table.png[Lens data table]
+
+Metric:: Display the latest number in a data series.
++
+[role="screenshot"]
+image::images/lens_metric.png[Lens metric]
+
+[float]
+[[create-lens-visualization]]
+=== Create Lens visualizations
+
+To create your Lens visualizations, use the visualization builder.
```

Comment:
I think this needs a little more context.  It quickly goes into changing the index.  What index am I using at the start?

Reply:
This is covered in the previous https://www.elastic.co/guide/en/kibana/current/createvis.html topic, but this could use some rearranging.